lib/Jsrt/ChakraCore.h

```diff
@@ -1063,5 +1063,22 @@ CHAKRA_API
         _Out_opt_ JsValueRef* target,
         _Out_opt_ JsValueRef* handler);
 
+///     Retrieves the <c>int64_t</c> value of a number value.
+/// </summary>
+/// <remarks>
```

Comment:
Nit: missing opening <summary> tag

Reply:
Thanks!  I must have dropped it when I was rebasing.